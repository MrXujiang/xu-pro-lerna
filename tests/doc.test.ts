import demoTest from './demo';

demoTest('docs');
