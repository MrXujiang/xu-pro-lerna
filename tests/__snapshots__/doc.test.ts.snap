// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`docs demos üì∏ renders ./docs/demos/customization-value-type.tsx correctly 1`] = `
Array [
  <div
    class="ant-pro-card ant-pro-card-border"
    style="margin-bottom: 24px;"
  >
    <div
      class="ant-pro-card-header"
    >
      <div
        class="ant-pro-card-title"
      >
        SchemaForm
      </div>
      <div
        class="ant-pro-card-extra"
      />
    </div>
    <div
      class="ant-pro-card-body"
    >
      <form
        autocomplete="off"
        class="ant-form ant-form-vertical"
        title="Ëá™ÂÆö‰πâ valueType"
      >
        <input
          style="display: none;"
          type="text"
        />
        <div
          class="ant-pro-form-group"
        >
          <div
            class="ant-pro-form-group-title"
          >
            Ê†áÁ≠æ
          </div>
          <div
            class="ant-space ant-space-horizontal ant-space-align-start ant-pro-form-group-container"
            style="row-gap: 0;"
          >
            <div
              class="ant-space-item"
              style="margin-right: 32px;"
            >
              <div
                class="ant-row ant-form-item"
              >
                <div
                  class="ant-col ant-form-item-label"
                >
                  <label
                    class=""
                    for="name"
                    title="Âè™ËØªÈìæÊé•"
                  >
                    Âè™ËØªÈìæÊé•
                  </label>
                </div>
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <a>
                        TradeCode 1
                      </a>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-space-item"
            >
              <div
                class="ant-row ant-form-item"
              >
                <div
                  class="ant-col ant-form-item-label"
                >
                  <label
                    class=""
                    for="name"
                    title="ÈìæÊé•"
                  >
                    ÈìæÊé•
                  </label>
                </div>
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <input
                        class="ant-input"
                        id="name"
                        placeholder="ËØ∑ËæìÂÖ•ÈìæÊé•"
                        type="text"
                        value="TradeCode 1"
                      />
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-pro-form-group"
        >
          <div
            class="ant-pro-form-group-title"
          >
            Ë∑ØÂæÑ
          </div>
          <div
            class="ant-space ant-space-horizontal ant-space-align-start ant-pro-form-group-container"
            style="row-gap: 0;"
          >
            <div
              class="ant-space-item"
              style="margin-right: 32px;"
            >
              <div
                class="ant-row ant-form-item"
              >
                <div
                  class="ant-col ant-form-item-label"
                >
                  <label
                    class=""
                    for="status"
                    title="Ê†áÁ≠æ"
                  >
                    Ê†áÁ≠æ
                  </label>
                </div>
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <div
                        class="ant-space ant-space-horizontal ant-space-align-center"
                      >
                        <div
                          class="ant-space-item"
                          style="margin-right: 8px;"
                        >
                          <span
                            class="ant-tag"
                          >
                            close
                          </span>
                        </div>
                        <div
                          class="ant-space-item"
                          style="margin-right: 8px;"
                        >
                          <span
                            class="ant-tag"
                          >
                            close
                          </span>
                        </div>
                        <div
                          class="ant-space-item"
                        >
                          <input
                            class="ant-input ant-input-sm"
                            style="width: 78px;"
                            type="text"
                            value=""
                          />
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="ant-space-item"
            >
              <div
                class="ant-row ant-form-item"
              >
                <div
                  class="ant-col ant-form-item-label"
                >
                  <label
                    class=""
                    for="status"
                    title="Âè™ËØªÊ†áÁ≠æ"
                  >
                    Âè™ËØªÊ†áÁ≠æ
                  </label>
                </div>
                <div
                  class="ant-col ant-form-item-control"
                >
                  <div
                    class="ant-form-item-control-input"
                  >
                    <div
                      class="ant-form-item-control-input-content"
                    >
                      <span
                        class="ant-tag"
                      >
                        close
                      </span>
                      <span
                        class="ant-tag"
                      >
                        close
                      </span>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-space ant-space-horizontal ant-space-align-center"
        >
          <div
            class="ant-space-item"
            style="margin-right: 8px;"
          >
            <button
              class="ant-btn ant-btn-primary"
              type="button"
            >
              <span>
                Êèê ‰∫§
              </span>
            </button>
          </div>
          <div
            class="ant-space-item"
          >
            <button
              class="ant-btn"
              type="button"
            >
              <span>
                Èáç ÁΩÆ
              </span>
            </button>
          </div>
        </div>
      </form>
    </div>
  </div>,
  <div
    class="ant-pro-card ant-pro-card-border"
    style="margin-bottom: 24px;"
  >
    <div
      class="ant-pro-card-header"
    >
      <div
        class="ant-pro-card-title"
      >
        ProTable
      </div>
      <div
        class="ant-pro-card-extra"
      />
    </div>
    <div
      class="ant-pro-card-body"
    >
      <div
        class="ant-pro-table"
      >
        <div
          class="ant-pro-table-search ant-pro-table-search-query-filter"
        >
          <form
            autocomplete="off"
            class="ant-form ant-form-horizontal ant-pro-form-query-filter"
          >
            <input
              style="display: none;"
              type="text"
            />
            <div
              class="ant-row ant-row-start"
              style="margin-left: -12px; margin-right: -12px;"
            >
              <div
                class="ant-col ant-col-8"
                style="padding-left: 12px; padding-right: 12px;"
              >
                <div
                  class="ant-row ant-form-item"
                >
                  <div
                    class="ant-col ant-form-item-label"
                    style="flex: 0 0 80px;"
                  >
                    <label
                      class=""
                      for="name"
                      title="ÈìæÊé•"
                    >
                      ÈìæÊé•
                    </label>
                  </div>
                  <div
                    class="ant-col ant-form-item-control"
                  >
                    <div
                      class="ant-form-item-control-input"
                    >
                      <div
                        class="ant-form-item-control-input-content"
                      >
                        <input
                          class="ant-input"
                          id="name"
                          placeholder="ËØ∑ËæìÂÖ•ÈìæÊé•"
                          style="width: 100%;"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="ant-col ant-col-8"
                style="padding-left: 12px; padding-right: 12px;"
              >
                <div
                  class="ant-row ant-form-item"
                >
                  <div
                    class="ant-col ant-form-item-label"
                    style="flex: 0 0 80px;"
                  >
                    <label
                      class=""
                      for="status"
                      title="Ê†áÁ≠æ"
                    >
                      Ê†áÁ≠æ
                    </label>
                  </div>
                  <div
                    class="ant-col ant-form-item-control"
                  >
                    <div
                      class="ant-form-item-control-input"
                    >
                      <div
                        class="ant-form-item-control-input-content"
                      >
                        <div
                          class="ant-space ant-space-horizontal ant-space-align-center"
                        >
                          <div
                            class="ant-space-item"
                          >
                            <input
                              class="ant-input ant-input-sm"
                              style="width: 78px;"
                              type="text"
                              value=""
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="ant-col ant-col-8"
                style="padding-left: 12px; padding-right: 12px; text-align: right;"
              >
                <div
                  class="ant-row ant-form-item pro-form-query-filter-actions"
                >
                  <div
                    class="ant-col ant-form-item-label"
                  >
                    <label
                      class="ant-form-item-no-colon"
                      title=" "
                    >
                       
                    </label>
                  </div>
                  <div
                    class="ant-col ant-form-item-control"
                  >
                    <div
                      class="ant-form-item-control-input"
                    >
                      <div
                        class="ant-form-item-control-input-content"
                      >
                        <div
                          class="ant-space ant-space-horizontal ant-space-align-center"
                        >
                          <div
                            class="ant-space-item"
                          >
                            <div
                              class="ant-space ant-space-horizontal ant-space-align-center"
                            >
                              <div
                                class="ant-space-item"
                                style="margin-right: 8px;"
                              >
                                <button
                                  class="ant-btn"
                                  type="button"
                                >
                                  <span>
                                    Èáç ÁΩÆ
                                  </span>
                                </button>
                              </div>
                              <div
                                class="ant-space-item"
                              >
                                <button
                                  class="ant-btn ant-btn-primary"
                                  type="button"
                                >
                                  <span>
                                    Êü• ËØ¢
                                  </span>
                                </button>
                              </div>
                            </div>
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </form>
        </div>
        <div
          class="ant-card"
        >
          <div
            class="ant-card-body"
            style="padding: 0px;"
          >
            <div
              class="ant-table-wrapper"
            >
              <div
                class="ant-spin-nested-loading"
              >
                <div
                  class="ant-spin-container"
                >
                  <div
                    class="ant-table ant-table-middle"
                  >
                    <div
                      class="ant-table-container"
                    >
                      <div
                        class="ant-table-content"
                      >
                        <table
                          style="table-layout: auto;"
                        >
                          <colgroup />
                          <thead
                            class="ant-table-thead"
                          >
                            <tr>
                              <th
                                class="ant-table-cell"
                              >
                                ÈìæÊé•
                              </th>
                              <th
                                class="ant-table-cell"
                              >
                                Ê†áÁ≠æ
                              </th>
                            </tr>
                          </thead>
                          <tbody
                            class="ant-table-tbody"
                          >
                            <tr
                              class="ant-table-row ant-table-row-level-0"
                              data-row-key="1"
                            >
                              <td
                                class="ant-table-cell"
                              >
                                <a>
                                  TradeCode 1
                                </a>
                              </td>
                              <td
                                class="ant-table-cell"
                              >
                                <span
                                  class="ant-tag"
                                >
                                  close
                                </span>
                                <span
                                  class="ant-tag"
                                >
                                  close
                                </span>
                              </td>
                            </tr>
                          </tbody>
                        </table>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <div
    class="ant-pro-card ant-pro-card-border"
    style="margin-bottom: 24px;"
  >
    <div
      class="ant-pro-card-header"
    >
      <div
        class="ant-pro-card-title"
      >
        ProDescriptions
      </div>
      <div
        class="ant-pro-card-extra"
      />
    </div>
    <div
      class="ant-pro-card-body"
    >
      <div
        class="ant-descriptions ant-pro-descriptions"
      >
        <div
          class="ant-descriptions-view"
        >
          <table>
            <tbody>
              <tr
                class="ant-descriptions-row"
              >
                <td
                  class="ant-descriptions-item"
                  colspan="1"
                >
                  <div
                    class="ant-descriptions-item-container"
                  >
                    <span
                      class="ant-descriptions-item-label"
                    >
                      ÈìæÊé•
                    </span>
                    <span
                      class="ant-descriptions-item-content"
                    >
                      <a>
                        TradeCode 1
                      </a>
                    </span>
                  </div>
                </td>
                <td
                  class="ant-descriptions-item"
                  colspan="2"
                >
                  <div
                    class="ant-descriptions-item-container"
                  >
                    <span
                      class="ant-descriptions-item-label"
                    >
                      Ê†áÁ≠æ
                    </span>
                    <span
                      class="ant-descriptions-item-content"
                    >
                      <span
                        class="ant-tag"
                      >
                        close
                      </span>
                      <span
                        class="ant-tag"
                      >
                        close
                      </span>
                    </span>
                  </div>
                </td>
              </tr>
            </tbody>
          </table>
        </div>
      </div>
    </div>
  </div>,
]
`;

exports[`docs demos üì∏ renders ./docs/demos/valueType.tsx correctly 1`] = `
Array [
  <div
    class="ant-row ant-form-item"
  >
    <div
      class="ant-col ant-form-item-label"
    >
      <label
        class=""
        title="valueType ÈÄâÊã©"
      >
        valueType ÈÄâÊã©
      </label>
    </div>
    <div
      class="ant-col ant-form-item-control"
    >
      <div
        class="ant-form-item-control-input"
      >
        <div
          class="ant-form-item-control-input-content"
        >
          <div
            class="ant-select ant-pro-filed-search-select ant-select-single ant-select-allow-clear ant-select-show-search"
            style="width: 200px;"
          >
            <div
              class="ant-select-selector"
            >
              <span
                class="ant-select-selection-search"
              >
                <input
                  aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
                  aria-autocomplete="list"
                  aria-controls="rc_select_TEST_OR_SSR_list"
                  aria-haspopup="listbox"
                  aria-owns="rc_select_TEST_OR_SSR_list"
                  autocomplete="off"
                  class="ant-select-selection-search-input"
                  id="rc_select_TEST_OR_SSR"
                  role="combobox"
                  type="search"
                  value=""
                />
              </span>
              <span
                class="ant-select-selection-item"
                title="ÊñáÊú¨Ê°Ü"
              >
                ÊñáÊú¨Ê°Ü
              </span>
            </div>
            <span
              aria-hidden="true"
              class="ant-select-clear"
              style="user-select: none;"
              unselectable="on"
            >
              <span
                aria-label="close-circle"
                class="anticon anticon-close-circle"
                role="img"
              >
                <svg
                  aria-hidden="true"
                  data-icon="close-circle"
                  fill="currentColor"
                  focusable="false"
                  height="1em"
                  viewBox="64 64 896 896"
                  width="1em"
                >
                  <path
                    d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
                  />
                </svg>
              </span>
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>,
  <form
    autocomplete="off"
    class="ant-form ant-form-vertical"
    trigger="[object Object]"
  >
    <input
      style="display: none;"
      type="text"
    />
    <div
      class="ant-pro-form-group"
    >
      <div
        class="ant-pro-form-group-title"
      >
        ÂàÜÁªÑ
      </div>
      <div
        class="ant-space ant-space-horizontal ant-space-align-start ant-pro-form-group-container"
        style="row-gap: 0;"
      >
        <div
          class="ant-space-item"
          style="margin-right: 32px;"
        >
          <div
            class="ant-row ant-form-item"
          >
            <div
              class="ant-col ant-form-item-label"
            >
              <label
                class="ant-form-item-required"
                for="text"
                title="ÁºñËæëÂô®"
              >
                ÁºñËæëÂô®
              </label>
            </div>
            <div
              class="ant-col ant-form-item-control"
            >
              <div
                class="ant-form-item-control-input"
              >
                <div
                  class="ant-form-item-control-input-content"
                >
                  <div
                    class="ant-select ant-pro-filed-search-select pro-field pro-field-m ant-select-single ant-select-allow-clear ant-select-show-arrow"
                    style="min-width: 100px;"
                  >
                    <div
                      class="ant-select-selector"
                    >
                      <span
                        class="ant-select-selection-search"
                      >
                        <input
                          aria-activedescendant="text_list_0"
                          aria-autocomplete="list"
                          aria-controls="text_list"
                          aria-haspopup="listbox"
                          aria-owns="text_list"
                          autocomplete="off"
                          class="ant-select-selection-search-input"
                          id="text"
                          readonly=""
                          role="combobox"
                          style="opacity: 0;"
                          type="search"
                          unselectable="on"
                          value=""
                        />
                      </span>
                      <span
                        class="ant-select-selection-item"
                        title="123456"
                      >
                        123456
                      </span>
                    </div>
                    <span
                      aria-hidden="true"
                      class="ant-select-arrow"
                      style="user-select: none;"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-select-suffix"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      aria-hidden="true"
                      class="ant-select-clear"
                      style="user-select: none;"
                      unselectable="on"
                    >
                      <span
                        aria-label="close-circle"
                        class="anticon anticon-close-circle"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="close-circle"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm165.4 618.2l-66-.3L512 563.4l-99.3 118.4-66.1.3c-4.4 0-8-3.5-8-8 0-1.9.7-3.7 1.9-5.2l130.1-155L340.5 359a8.32 8.32 0 01-1.9-5.2c0-4.4 3.6-8 8-8l66.1.3L512 464.6l99.3-118.4 66-.3c4.4 0 8 3.5 8 8 0 1.9-.7 3.7-1.9 5.2L553.5 514l130 155c1.2 1.5 1.9 3.3 1.9 5.2 0 4.4-3.6 8-8 8z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="ant-space-item"
        >
          <div
            class="ant-row ant-form-item"
          >
            <div
              class="ant-col ant-form-item-label"
            >
              <label
                class=""
                for="text"
                title="Âè™ËØª"
              >
                Âè™ËØª
              </label>
            </div>
            <div
              class="ant-col ant-form-item-control"
            >
              <div
                class="ant-form-item-control-input"
              >
                <div
                  class="ant-form-item-control-input-content"
                >
                  123456
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="ant-space ant-space-horizontal ant-space-align-center"
    >
      <div
        class="ant-space-item"
        style="margin-right: 8px;"
      >
        <button
          class="ant-btn ant-btn-primary"
          type="button"
        >
          <span>
            Êèê ‰∫§
          </span>
        </button>
      </div>
      <div
        class="ant-space-item"
      >
        <button
          class="ant-btn"
          type="button"
        >
          <span>
            Èáç ÁΩÆ
          </span>
        </button>
      </div>
    </div>
  </form>,
]
`;
